examples/biobert_ner/bio_ner_predictor.py

```diff
@@ -165,32 +190,36 @@ def pack(self, data_pack: DataPack,
         ner to the original subwords and convert predictions to something that
         makes sense in a word-by-word segmentation
         """
-
+        # print(list(data_pack.get(Sentence)))
         if output_dict is None:
             return
 
         for i in range(len(output_dict["Subword"]["tid"])):
             tids = output_dict["Subword"]["tid"][i]
             labels = output_dict["Subword"]["ner"][i]
 
-            # Filter to labels not in `self.ignore_labels`
+            entities = []
+            # Filter to labels not in `self.ft_configs.ignore_labels`
             entities = [dict(idx=idx, label=label, tid=tid)
                         for idx, (label, tid) in enumerate(zip(labels, tids))
                         if label not in self.ft_configs.ignore_labels]
 
-            entity_groups = self._group_entities(entities, data_pack, tids)
+            entity_groups = self._compose_entities(entities, data_pack, tids)
             # Add NER tags and create EntityMention ontologies.
-            for first_tid, last_tid in entity_groups:
+            for first_idx, last_idx in entity_groups:
                 first_token: Subword = data_pack.get_entry(  # type: ignore
-                    first_tid)
+                    tids[first_idx])
                 first_token.ner = 'B-' + self.ft_configs.ner_type
```

Comment:
We can make it clear in the readme why we only have one ner type in this example.

Reply:
OK